test/__snapshots__/type-generator.test.ts.snap

```diff
@@ -302,11 +352,11 @@ export function toJson_TestType(obj: TestType | undefined): Record<string, any>
  */
 export enum FqnOfTestTypePercentiles {
   /** 0.9 */
-  VALUE_0_9 = 0.9,
+  VALUE_0_PERIOD = 0.9,
   /** 0.95 */
-  VALUE_0_95 = 0.95,
+  VALUE_0_PERIOD_95 = 0.95,
   /** 0.99 */
-  VALUE_0_99 = 0.99,
+  VALUE_0_PERIOD_99 = 0.99,
```

Comment:
`Callout`: Looks like this would be a breaking change and would change enum names in certain scenarios.

Reply:
Oh good catch, that's probably not great.

Can we make an exception for a period inside numbers?